content/guides/authentication.md

```diff
@@ -8,17 +8,29 @@ weight: 2
 
 There are two sorts of keys in the dfuse ecosystem:
 
-1. A long-lived API key, which looks like `server_abcdef123123123000000000000000000`, used to get short-lived JWT.
+1. A long-lived API key, which will resemble `server_abcdef123123123000000000000000000`, used to generate short-lived <a href="https://jwt.io/" title="JSON Web Tokens">JWT</a>.
+2. A short-lived JWT, used when performing any call on the dfuse Platform, which will resemble: `eyJhbGciOiJLTVNFUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE1NTYxMzI4MjAsImp0aSI6IjQwNWVmOTUxLTAwZTYtNGJmNC1hZWMxLTU0NTU1ZWMzMTUwMiIsImlhdCI6MTU1NjA0NjQyMCwiaXNzIjoiZGZ1c2UuaW8iLCJzdWIiOiJ1aWQ6MHdlbnU2NmUwNzU4OWRhODY4MWNlIiwiYWtpIjoiM2NhYWEzYzA3M2FlZjVkMmYxOGUwNjJmZDkzYzg3YzMzYWIxYzA1YzEzNjI3NjU2OTgzN2Y5NDc5NzZlMjM0YSIsInRpZXIiOiJmcmVlLXYxIiwic3RibGsiOi0zNjAwLCJ2IjoxfQ.000HeTujIuS_LRvvPN6ZRCmtoZqZyV6P1enNBviwK8v7Tf7BLHJIrEpQoEREKSIMdZWPrMQl_OE55yJP0MxUDA`
 
-2. A short-lived JWT, used to do any call on the dfuse Platform, which looks like: `eyJhbGciOiJLTVNFUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE1NTYxMzI4MjAsImp0aSI6IjQwNWVmOTUxLTAwZTYtNGJmNC1hZWMxLTU0NTU1ZWMzMTUwMiIsImlhdCI6MTU1NjA0NjQyMCwiaXNzIjoiZGZ1c2UuaW8iLCJzdWIiOiJ1aWQ6MHdlbnU2NmUwNzU4OWRhODY4MWNlIiwiYWtpIjoiM2NhYWEzYzA3M2FlZjVkMmYxOGUwNjJmZDkzYzg3YzMzYWIxYzA1YzEzNjI3NjU2OTgzN2Y5NDc5NzZlMjM0YSIsInRpZXIiOiJmcmVlLXYxIiwic3RibGsiOi0zNjAwLCJ2IjoxfQ.000HeTujIuS_LRvvPN6ZRCmtoZqZyV6P1enNBviwK8v7Tf7BLHJIrEpQoEREKSIMdZWPrMQl_OE55yJP0MxUDA`
+## Issuing a Long-Lived API Key
+
+Once you have created an account through the <a href="https://app.dfuse.io" title="dfuse API Portal">dfuse portal</a> you will be able to generate an API key. 
+
+Click on "GENERATE NEW KEY" and you'll now be able to start customizing your key for your specific needs.
+First assign the API key a name so that you'll be able to differentiate it from other API keys that you generate.
+Then, select the key-type between `mobile`, `web`, or `server`.
+  * `Mobile` - adaptive rate limiting to ensure abuse prevention. The rate limiting thresholds will eventually be controllable by yourself. `Mobile` can handle a large number of IPs, with more granular rate limiting by IP.
+  * `Web` - adaptive rate limiting to ensure abuse prevention. The rate limiting thresholds will eventually be controllable by yourself. JWTs generated by a `Web` key will be scoped to a small number of IPs, with heavy limiting of calls per IP. `Web` keys will have their `Origin header` verified to help with abuse prevention.
+  * `Server` - Should be used for server-to-server data access. Aggressive rate limiting for JWT issuance, as the generated tokens should be cached and used for their lifespan. Accepts a large number of calls per IP.
+  
+For ease of identification, all keys will be prefixed by their key type. `Mobile` and `web` keys are safe to publish within your mobile or web application.
 
 ## Issuing a Short-Lived JWT
```

Comment:
Should we simplify the titles to "Issuing an API Key" and "Issuing a JWT"?

Also, I'm not sure issuing is the right word. The user doesn't issue the key - we do. The user `obtains` or `gets` a key.

Reply:
Agreed. I would use "Obtain an API Key" and "Obtain a JWT"